src/brain/issueLabelHandler/index.test.ts

```diff
@@ -159,6 +168,7 @@ describe('issueLabelHandler', function () {
       {
         ...makePayload(repo, undefined, username),
         comment: { author_association: membership },
+        ...(isPR ? { issue: { pull_request: {} } } : {}),
```

Comment:
nit: I *think* I get what you're doing here ("only if`isPR === true` add the field `issue: { pull_request: {} }` to the object I am building"), but its pretty hard to read in practice.